autoconsent/autoconsent-impl/src/test/java/com/duckduckgo/autoconsent/impl/RealAutoconsentTest.kt

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autoconsent.impl
+
+import android.webkit.WebView
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.anyOrNull
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+
+class RealAutoconsentTest {
+
+    private val pluginPoint = FakePluginPoint()
+    private val repository = FakeRepository()
+    private val mockWebView: WebView = mock()
+
+    lateinit var testee: RealAutoconsent
```

Comment:
nit: never told anybody but I hate the `testee` name. It forces me to go and check what's the type. I generally prefer to name variables after the type, ie. `realAutoconsent` or `autoconsent` in this case

Reply:
Yeah I can agree with that :D